### DISCUSSION TITLE
Link from subdomain-folder of YOURLS to main domain with CORS issue

### DISCUSSION THREAD

**Discussion by areopagites** [YOURLS/YOURLS#2979] [Q&A]
I would like to create a page like this with YOURLS functionality:
http://yourwish.es

But I have a subfolder like "https://example.com/urls/"
Is it possible to get access from the main domain "example.com" to the subfolder "/urls/" with YOURLS?
I have copied the url bar and the "add_link()"-button to the html-website on the main domain.
All script sources link to the subfolder like this:
`<script src="https://example.com/urls/js/common.js?v=1.7.9" type="text/javascript"></script>`

On my website all browsers ignored the click on the "add_link()"-button at the main domain, but when I try it on "https://example.com/urls/admin" everything works.
So I try to solve the CORS-issue with a line in config.php:
`<?php
header("Access-Control-Allow-Origin: http://example.com Vary: Origin");`

And in functions-html.php:
`function yourls_content_type_header( $type ) {
    yourls_do_action( 'content_type_header', $type );
	if( !headers_sent() ) {
		$charset = yourls_apply_filter( 'content_type_header_charset', 'utf-8' );
		header( "Content-Type: $type; charset=$charset" );
		header("Access-Control-Allow-Origin: YOURLS_SITE");
		return true;
	}
	return false;
}`
But nothing happens.
What can I do?

**Comment by dgw** [comment]
Is this actually a CORS issue?

Needs more information like version numbers, httpd, platform, browser, and most importantly logs from server & browser console.

**Comment by areopagites** [comment]
![Screenshot 2020-12-28 20 55 32](https://user-images.githubusercontent.com/16578349/103240040-80f95500-494f-11eb-98e2-ea37c01eb5cd.png)
Only the jquery.cal.js with the undefined variable "l10n_cal_month" is listed as an issue in the browser console, now.
But I think it would also work without this variable.
Before I fixed the CORS issue, this was the result of the console:
![ACAO](https://user-images.githubusercontent.com/16578349/103240217-ff55f700-494f-11eb-8fe4-4fd51483c4fa.png)
You can see the result of my main website: https://fancythatshit.com
But the button doesnt work.
I try it with the latest versions of Chrome, IE and Opera.
Webhosting-Server: Debian, Version "Buster"
Webserver: nginx and Apache
